pkgs/by-name/op/openscap/package.nix

```diff
@@ -139,7 +134,7 @@ stdenv.mkDerivation rec {
   ];
 
   postBuild = ''
-    make docs
+    make $makeFlags docs
   '';
 
   installPhase = ''
```

Comment:
Consider using `postInstall` instead of calling `make install`.

Reply:
This is unrelated to this PR, please create a separated PR if you want to change this.